### DISCUSSION TITLE
Prevent cut tool from dissecting a polygon?

### DISCUSSION THREAD

**Discussion by roderickn-wt** [geoman-io/leaflet-geoman#1034] [Questions & Help]
I am starting here cause I am not sure if this is even an option that I can see, but is there a way to prevent the cut tool from dissecting a polygon into two? I have read through the options available and unless I don't understand what one of the options is correct I am not seeing a way to prevent dissecting.

**Comment by Falke-Design** [comment]
You can add to the layer the option `allowCutting: false`.

```
layer.pm.setOptions({allowCutting: false})
```

**Reply by roderickn-wt** [reply]
This block cutting of the polygon entirely. I am looking to allow cutting holes, notches, corners, but I don't that the cut to slice the shape into two pieces.

**Reply by Falke-Design** [reply]
No sorry, that is not possible

**Reply by roderickn-wt** [reply]
ok, thank you.
